.github/workflows/nf-test.yml

```diff
@@ -78,7 +78,7 @@ jobs:
           - isMain: false
             profile: "singularity"
         NXF_VER:
-          - "24.10.5"
+          - "25.04.8"
```

Comment:
Is it appropriate to bump a major Nextflow version 24 &rarr; 25 in a dot release x.x.0 &rarr; x.x.1 of a pipeline? If so, it needs to be prominent in the release notes and other announcements for this release